docs/libraries.md

```diff
@@ -1,24 +1,26 @@
-# Libraries 
+# Libraries
 
 ## Overview
 This document is an overview of Armada client libraries.
 
 ## Python
-Here, we give an overview of the Armada Python client. 
+Here, we give an overview of the Armada Python client.
 
 ### Installation
 Prerequisites for installation include a working Python installation and poetry. You may wish to install the client into a virutalenv.
 
 Our python client bindings can be installed directly from the Armada repository.
  1. Checkout the [armada git repository](https://github.com/g-research/armada)
  2. From the root of the repository, run `make python` to generated needed gRPC code.
- 3. Install the client with poetry: `cd client/python && poetry install -v` 
+ 3. Install the client with poetry: `cd client/python && poetry install -v`
```

Comment:
I don't think you actually changed this line but this calls out that we need to update Libraries.md with the latest building changes.  

@jayofdoom do you think we should just link to README.md or contributor.md in our docs so we don't need to maintain both?

Reply:
I would agree that just linking to README.md and removing this stuff would be wise. I will hold off for now though.

Reply:
Yeah. Literally consider all the existing stuff in the libraries.md boilerplate that can/should be replaced with internal links and/or improved information.

Reply:
Cool. I shall make those changes :))